net/snode.c/Config.in

```diff
@@ -0,0 +1,344 @@
+menu "Configuration"
+	depends on PACKAGE_snode.c-utils
+
+	config SNODEC_GROUP_ID
+		int
+		prompt "Group id of group snodec"
+		default "1023"
+		help
+				The group id for unix group 'snodec'. This group is used for config/log/pid file
```

Comment:
This looks like complicated. Why you can not create such name and user in Makefile?

Reply:
Yes, you are right - this is obviously to complicate. I don't know how often I have read the "Creating packages" documentation but never "realized" the "USERID" variable. Now I have found it and made use of it. Nevertheless this doesn't remove the "user" and "group" in case the package gets uninstalled. Is this intended?